ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -2334,7 +2344,6 @@ static int zend_jit_hybrid_loop_hot_counter_stub(dasm_State **Dst)
 	}
 
 	|->hybrid_loop_hot_counter:
-
```

Comment:
You probably missed  ENDBR here.
Personally, Now I would move ENDBR into ``zend_jit_hybrid_hot_counter_stub()`` and ``zend_jit_hybrid_trace_counter_stub()``. I don't see any other problems.
Did you run tests? Everything is passed?
Please fix the last issue, and merge this or let me do this.

Reply:
> Now I would move ENDBR into `zend_jit_hybrid_hot_counter_stub()` and `zend_jit_hybrid_trace_counter_stub()`

Done and it looks good.

**Test: Passed**
On a full CET environment/stack at CET enabled Intel CPU:
JIT test like `make test TESTS="ext/opcache/tests/jit"` passed. Without the patch, the test fails.
This patch doesn't incur addition fail in `make test`.

I think I don't have permission to merge?